packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -78,6 +78,9 @@ function registerCommands(): vscode.Disposable {
 export function activate(context: vscode.ExtensionContext) {
   console.log('SFDX CLI Extension Activated');
 
+  // Context
+  vscode.commands.executeCommand('setContext', 'sfdx:project_opened', true);
```

Comment:
This technique is from https://github.com/Microsoft/vscode/issues/10471#issuecomment-239764839